src/dialogs/more-info/controls/more-info-climate.ts

```diff
@@ -394,7 +394,7 @@ class MoreInfoClimate extends LitElement {
       [key: string]: unknown;
     }
   ) {
-    if (oldVal === newVal) {
+    if (oldVal === newVal || !newVal) {
```

Comment:
This is now for every service call, this also means you can no longer set the temperature to `0` for example.

This guard should be in `_handlePresetmodeChanged`